providers/src/airflow/providers/fab/www/templates/airflow/main.html

```diff
@@ -0,0 +1,85 @@
+{#
+ Licensed to the Apache Software Foundation (ASF) under one
+ or more contributor license agreements.  See the NOTICE file
+ distributed with this work for additional information
+ regarding copyright ownership.  The ASF licenses this file
+ to you under the Apache License, Version 2.0 (the
+ "License"); you may not use this file except in compliance
+ with the License.  You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing,
+ software distributed under the License is distributed on an
+ "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ KIND, either express or implied.  See the License for the
+ specific language governing permissions and limitations
+ under the License.
+#}
+
+{% extends 'appbuilder/baselayout.html' %}
+{% from 'airflow/_messages.html' import show_message %}
+
+{% block page_title -%}
+  Airflow - Airflow 2 plugins compatibility view
+{% endblock %}
+
+{% block head_css %}
+  {{ super() }}
+
+  {% if not appbuilder.app_theme %}
+    {# airflowDefaultTheme.css file contains the styles from local bootstrap-theme.css #}
+    <link rel="stylesheet" type="text/css" href="{{ url_for_asset('airflowDefaultTheme.css') }}">
+  {% endif %}
+  <link rel="stylesheet" type="text/css" href="{{ url_for_asset('materialIcons.css') }}">
+  <link rel="stylesheet" type="text/css" href="{{ url_for_asset('main.css') }}">
+  <link rel="stylesheet" type="text/css" href="{{ url_for_asset('loadingDots.css') }}">
+  <link rel="stylesheet" type="text/css" href="{{ url_for_asset('jquery-ui.min.css') }}">
+  <style type="text/css">
+    {% for state, state_color in state_color_mapping.items() %}
+      span.{{state}} {
+        background-color: {{state_color}};
+      }
+    {% endfor %}
+    .navbar-nav > li > a {
+      color: {{ navbar_text_color }};
+    }
+    .navbar-nav > li > a:hover {
+      background-color:  {{ navbar_hover_color }};
+      color: {{ navbar_text_hover_color }};
+    }
+  </style>
+  <link rel="icon" type="image/png" href="{{ url_for('static', filename='pin_32.png') }}">
+{% endblock %}
+
+{% block messages %}
+  {% call show_message(category='warning', dismissible=false) %}
+    <p>
+      This is a limited version of an Airflow 2 environment. The reason why you are using this environment is that
+      you are accessing a page defined in an Airflow plugin only compatible with Airflow 2.
+      This is also why the UI is different from the rest of your Airflow environment.
+    </p>
+    <p>
+      To have the plugin be integrated with the rest of your environment,
+      please update your plugin to make it compatible with Airflow 3.
```

Comment:
```suggestion
      You have a plugin that is using a FAB view or Flask Blueprint, which was used for the Airflow 2 UI, and is now
      deprecated. Please update your plugin to be compatible with the Airflow 3 UI.
```

Maybe something like this?

We probably need to revisit anyways, because we likely need to leave this UI around for the FAB auth manager too - we we may need this behind a conditional on plugin stuff. But this is a problem for later :)